src/ASECryptoStream.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
```

Comment:
Please provide your authorship details or original authors and source of the code (if any)